backend/curriculum_tracking/tests/test_activity_log_entry_creators.py

```diff
@@ -122,6 +122,7 @@ def test_start_project(self):
                 project_submission_type=ContentItem.LINK, template_repo=None
             ),
         )
+        creators.EventType.objects.get_or_create(name=creators.CARD_MOVED_TO_REVIEW_FEEDBACK)
```

Comment:
These tests shouldn't need to be updated.

If you make your sql queries more efficient then these changes wont be required.